hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -95,28 +95,6 @@ protected NearCacheConfig createNearCacheConfig(InMemoryFormat inMemoryFormat) {
                 .setInMemoryFormat(inMemoryFormat);
     }
 
-    protected class NearCacheTestContext {
```

Comment:
This class was refactored to top-level class as it is generally useful for near-cache tests.